release_notes_generator/utils/constants.py

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 #
 
+# pylint: disable=too-few-public-methods
```

Comment:
This should not be a class, as we discussed together earlier.

Reply:
Class removed.